src/game/scenes/main-scene/MainScene.js

```diff
@@ -631,6 +637,11 @@ export default class MainScene extends Phaser.Scene {
         self.allPlayers[responsePlayer.walletAddress]?.playHealEffect();
         break;
       }
+
+      case Const.Command.tokensSent: {
+        self.allPlayers = response.players;
+        break;
```

Comment:
i think this may be an issue as it may possibly happen that after the game is over MainScene will go to LeaderboardScene without receiving this message (then team bonus will not be shown in the Leaderboard)

Reply:
shouldn't the leaderboard simply ask for the latest state? (our CU has a new endpoint for this)